set-web/src/main/java/eu/europeana/set/web/search/UserSetQueryBuilder.java

```diff
@@ -18,10 +16,10 @@
 import eu.europeana.set.definitions.model.vocabulary.WebUserSetModelFields;
 
 public class UserSetQueryBuilder extends QueryBuilder {
-    
+
     String[] fields = new String[] {WebUserSetModelFields.CREATOR, WebUserSetModelFields.VISIBILITY,
-	    WebUserSetFields.TYPE, WebUserSetFields.ITEM}; 
-    Set<String> suportedFields = Set.of(fields);
+	    WebUserSetFields.TYPE, WebUserSetFields.ITEM, WebUserSetFields.SET_ID};
+    Set<String> suportedFields = new HashSet<>(Arrays.asList(fields));
```

Comment:
please revert this line of code as the java version was changes to 11

Reply:
done